staging/src/k8s.io/client-go/discovery/discovery_client.go

```diff
@@ -43,6 +44,13 @@ const (
 	mimePb = "application/com.github.proto-openapi.spec.v2@v1.0+protobuf"
 )
 
+var (
+	// DefaultTimeout is the maximum amount of time per request when no timeout has been set on a RESTClient.
+	// Defaults to 32s in order to have a distinguishable length of time, relative to other timeouts that exist.
+	// It's a variable to be able to change it in tests.
+	DefaultTimeout = 32 * time.Second
```

Comment:
can be private, right?

Reply:
I'd wish, the tests are explicitly using `discover_test` package to separate from this package, so I had to make public. I don't know the reasoning behind it, but I didn't want to change it without syncing with you about it.